scripts/get-scan-results.js

```diff
@@ -0,0 +1,22 @@
+import { getOctokit } from "@actions/github";
+
+const owner = "coveo";
+const repo = "semantic-monorepo-tools";
+const octokit = getOctokit(process.env.GITHUB_TOKEN);
+
+const getSnykCodeAlerts = () => {
+  return octokit.rest.codeScanning.listAlertsForRepo({
+    owner,
+    repo,
+    ref: "master",
```

Comment:
Shouldn't that be main?
```suggestion
    ref: "main",
```

Reply:
good catch!